secretcache/cacheItem.go

```diff
@@ -129,6 +129,12 @@ func (ci *secretCacheItem) getVersion(versionStage string) (*cacheVersion, bool)
 	return secretCacheVersion, true
 }
 
+// refresh the cached object on demand
+func (ci *secretCacheItem) refreshNow(ctx context.Context) {
```

Comment:
nit: indentation

Reply:
This is auto formatted with go fmt and seems like it matches the file, what am I missing?

Reply:
Yeah, I am not sure what that crazy person a few months was seeing. Looks good to me now ....